spec/forms/idv/profile_form_spec.rb

```diff
@@ -83,13 +84,31 @@
     end
   end
 
+  describe 'zipcode validity' do
+    it 'populates error for :zipcode when invalid' do
+      %w(1234 123Ac-1234 1234B).each do |invalid_zip|
+        subject.submit(profile_attrs.merge(zipcode: invalid_zip))
+        expect(subject.errors[:zipcode]).to eq [I18n.t('idv.errors.pattern_mismatch.zipcode')]
+      end
+    end
+  end
+
+  describe 'ssn validity' do
+    it 'populates errors for :ssn when invalid' do
+      %w(1234 123-1-1111 abc-11-1123).each do |invalid_ssn|
```

Comment:
I think, based on the above comments, we aren't going to require the `-`. I can definitely add additional test cases though if this seems incomplete!